Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Generic Artifacts Modifiers

#### Changelog:
- Changed balance of generic artifact in order to make staking modifiers less OP (no more 100% cost reduction from two rings, I hope)

#### How to test:
- Check that modifiers [here](https://docs.google.com/spreadsheets/d/1xDjRHfG-ovKVcEQVfv02BsK3j0dgwS2rfH7KJ4GirA0/edit#gid=857004952&range=1:168) and modifiers in-game are the same.
